src/browser/provider/index.ts

```diff
@@ -139,8 +142,20 @@ export default class BrowserProvider {
         await this.plugin.waitForConnectionReady(browserId);
         await delay(BROWSER_OPENING_DELAY);
 
-        if (this.localBrowsersInfo[browserId])
-            this.localBrowsersInfo[browserId].windowDescriptor = await browserTools.findWindow(browserId);
+        if (this.localBrowsersInfo[browserId]) {
+            const connection     = BrowserConnection.getById(browserId) as BrowserConnection;
+            let windowDescriptor = null;
+
+            try {
+                windowDescriptor = await browserTools.findWindow(browserId);
+            }
+            catch (err) {
+                DEBUG_LOGGER(err);
```

Comment:
```suggestion
                // NOTE: We can suppress the error here since we can just disable window manipulation functions
                // when we cannot find a local window descriptor
                DEBUG_LOGGER(err);
```